src/xbot/common/command/scripted/CommandScriptInterface.java

```diff
@@ -0,0 +1,79 @@
+package xbot.common.command.scripted;
+
+import org.apache.log4j.Logger;
+
+import edu.wpi.first.wpilibj.command.Command;
+import xbot.common.command.BaseCommand;
+
+/**
+ * An object which can be exposed to a running script to wrap a
+ * command on the scheduler.
+ *
+ */
+public class CommandScriptInterface {
+    static Logger log = Logger.getLogger(CommandScriptInterface.class);
+    
+    private Command wrappedCommand;
+    private ScriptedCommand parentCommand;
+
+    public CommandScriptInterface(Command wrappedCommand, ScriptedCommand parentCommand) {
+        this.wrappedCommand = wrappedCommand;
+        this.parentCommand = parentCommand;
```

Comment:
go ahead and add a "init" logging here - it's really nice to see in the logs to see what's getting initialized.